InscryptionAPI/Triggers/Interfaces.cs

```diff
@@ -1,240 +1,414 @@
     /// </summary>
-    /// <param name="originalSlots">The set of original card slots that the card would attack (as set by abilities in the base game)</param>
-    /// <param name="otherAddedSlots">Slots that have been added by other custom abilities</param>
-    /// <returns>The list of card slots you want to attack</returns>
-    /// <remarks>If your card is replacing the default attack slot (the opposing slot) see 'RemoveDefaultAttackSlot'
-    /// If you are **not** replacing the default attack slot, do **not** include that here. Simple ensure 'RemoveDefaultAttackSlot' returns false.
-    /// Only return the default attack slot if you want to attack it an additional time.</remarks>
-    List<CardSlot> GetOpposingSlots(List<CardSlot> originalSlots, List<CardSlot> otherAddedSlots);
+    public interface IOnOtherCardDealtDamageInHand
+    {
+        public bool RespondsToOtherCardDealtDamageInHand(PlayableCard attacker, int amount, PlayableCard target);
+        public IEnumerator OnOtherCardDealtDamageInHand(PlayableCard attacker, int amount, PlayableCard target);
+    }
 
     /// <summary>
-    /// If true, this means that the attack slots provided by GetOpposingSlots should override the default attack slot. 
-    /// If false, it means they should be in addition to the default opposing slot.
+    /// Trigger that is triggered after any card dies, but unlike normal OnOtherCardDie this one only works in hand.
     /// </summary>
-    bool RemoveDefaultAttackSlot();
-}
+    public interface IOnOtherCardDieInHand
+    {
+        public bool RespondsToOtherCardDieInHand(PlayableCard card, CardSlot deathSlot, bool fromCombat, PlayableCard killer);
+        public IEnumerator OnOtherCardDieInHand(PlayableCard card, CardSlot deathSlot, bool fromCombat, PlayableCard killer);
+    }
 
-/// <summary>
-/// Used when a card wants to provide a passive buff to other cards on the board
-/// </summary>
-public interface IPassiveAttackBuff
-{
     /// <summary>
-    /// Used to provide a passive attack buff to a target
+    /// Trigger that is triggered before any item is used.
     /// </summary>
-    /// <returns>The amount of attack you want to buff the target by</returns>
-    /// <remarks>Do not assume that the target is on your side of the board! There may be negative sigils that buff opposing cards.</remarks>
-    int GetPassiveAttackBuff(PlayableCard target);
-}
+    public interface IOnPreItemUsed
+    {
+        public bool RespondsToPreItemUsed(string itemname, bool isHammer);
+        public IEnumerator OnPreItemUsed(string itemname, bool isHammer);
+    }
+
+    /// <summary>
+    /// Trigger that is triggered after any item is used.
+    /// </summary>
+    public interface IOnPostItemUsed
+    {
+        public bool RespondsToPostItemUsed(string itemname, bool success, bool isHammer);
+        public IEnumerator OnPostItemUsed(string itemname, bool success, bool isHammer);
```

Comment:
Please camelCase and add some documenation. I know the name itself is straightforward, but will be nice to have consistency.